core/common/src/main/java/alluxio/worker/block/io/BlockWriter.java

```diff
@@ -37,4 +39,17 @@
    * @return channel
    */
   GatheringByteChannel getChannel();
+
+  /**
+   * Transfers buf.writableBytes() bytes to the this block writer from the given buf.
```

Comment:
writableBytes or readableBytes? it seems in the LocalFileBlockWriter and MockBlockWriter, we both use `readable` bytes